docs/advanced-topics/stack-monitoring.asciidoc

```diff
@@ -0,0 +1,160 @@
+:page_id: stack-monitoring
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= Stack Monitoring
+
+You can enable link:https://www.elastic.co/guide/en/elasticsearch/reference/current/monitor-elasticsearch-cluster.html[Stack Monitoring]
+on your Elasticsearch and Kibana to collect and ship their metrics and logs.
+
+You just have to reference a separate monitoring Elasticsearch cluster in the `spec.monitoring` section.
```

Comment:
```suggestion
To enable stack monitoring, simply reference the monitoring Elasticsearch cluster in the spec.monitoring section of the ECK configuration.
```

Reply:
`of the ECK configuration` is not really correct here. The ECK operator has a [configuration](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-operator-config.html). Below this sentence, the YAML code corresponds to the Elasticsearch cluster specification and the Kibana instance(s) specification.

As we just said ES & KB in "You can enable Stack Monitoring on your Elasticsearch and Kibana to collect and ship their metrics and logs.", can we say:

To enable stack monitoring, simply reference the monitoring Elasticsearch cluster in the `spec.monitoring` section of their specification.

Reply:
Thanks. I wasn't sure about that configuration part. Your suggested text looks perfect.